Makefile

```diff
@@ -3,7 +3,7 @@
 ENABLE_BRANCH_COVERAGE ?= 0
 AUTO_FIX_IMPORTS ?= 0
 
-ifneq ($(AUTO_FIX_IMPORTS), 1)
+ifneq ($(FIX), 1)
```

Comment:
I got tired of typing `AUTO_FIX_IMPORTS` whenever I wanted `make imports` to actually fix the problem rather than just complaining. Didn't intend to commit it, will revert.